sep490-idp/src/main/resources/application.yml

```
@@ -82,3 +85,10 @@ logging:
             sql: ${HIBERNATE_SQL_PARAM_LOG_LEVEL:TRACE}
 server:
   port: 8180
+  servlet:
+    context-path: /idp
+  forward-headers-strategy: native
+  tomcat:
+    remoteip:
+      remote-ip-header: x-forwarded-for
+      protocol-header: x-forwarded-proto
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)